apps/api/env/.env.functional.test

```
@@ -38,3 +38,6 @@ AMPLITUDE_SAMPLING=1
 NOTIFICATIONS_API_BASE_URL=http://localhost:3081
 
 FEATURE_FLAGS_ENABLE_ALL=true
+
+PROXY_API_URL=https://api.sandbox-01.aksh.pw:443
+PROXY_RPC_URL=https://rpc.sandbox-01.aksh.pw:443
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)